frontend/src/libs/components/profile-settings-sidebar/profile-settings-button/profile-settings-button.tsx

```diff
@@ -0,0 +1,41 @@
+import { Icon } from '#libs/components/components.js';
+import { type IconName } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/get-valid-class-names.js';
```

Comment:
```suggestion
import { getValidClassNames } from '#libs/helpers/helpers.js';
```

Reply:
@fshabanov Do you mean that we can pass color property to an icon component or you said about that we can set coolor for icon in css in wrapper element?

Reply:
@Xxandder we can pass `color` property to Icon component. I don't think it would be possible to change icon color using wrapper

Reply:
@fshabanov We can set color with css styles. it was disscussed [here](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/140#discussion_r1312694559). So we set fill="currentColor" in svg and this svg will have color as in parent element